eng/pipelines/templates/steps/build-test.yml

```diff
@@ -68,30 +68,30 @@ steps:
           }
         }
       displayName: 'Run Tests'
-      workingDirectory: '${{parameters.GoWorkspace}}'
+      workingDirectory: $(System.DefaultWorkingDirectory)
       env:
         GO111MODULE: 'on'
         PROXY_CERT: $(Build.SourcesDirectory)/eng/common/testproxy/dotnet-devcert.crt
         ${{ insert }}: ${{ parameters.EnvVars }}
 
     - pwsh: ../eng/scripts/create_coverage.ps1 ${{parameters.ServiceDirectory}}
       displayName: 'Generate Coverage XML'
-      workingDirectory: '${{parameters.GoWorkspace}}sdk'
+      workingDirectory: $(System.DefaultWorkingDirectory)/sdk
```

Comment:
We should look at remove the "sdk" path here and let the create_coverage script understand that is needed with a servicedirectory.